doc/irmin-pack/design/lower_layer.md

```diff
@@ -308,21 +308,21 @@ To successfully go from commit hash to data, we need to answer the following que
 - How do we determine which volume contains a commit hash?
 - Once we have loaded a commit, how do we continue reading its data (nodes and contents) from its volume?
 
-### Finding a commit
+### Finding a Commit
 
-**Upper layer**
+**Upper Layer**
 
-The starting point for finding a commit in `irmin-pack` is to lookup its key in the index. Assuming the store has a record in its index of the commit hash, the key will provide a global offset for finding the commit. Lets call this simply `offset`.
+The starting point for finding a commit in `irmin-pack` is to lookup its key in the index. Assuming the store has a record in its commit hash's index, the key will provide a global offset for finding the commit. Lets call this simply `offset`.
```

Comment:
```suggestion
The starting point for finding a commit in `irmin-pack` is to lookup its key in the index. Assuming the store has a record of the commit hash in its index, the key will provide a global offset for finding the commit. Lets call this simply `offset`.
```

I'm not sure what's the proper English way to formulate this sentence, so I tried switching words around... but don't hesitate if you have a better idea! (but the "of" is not a possessive)

Reply:
Let's just put it back the way it was, then. 🙂